src/interval.jl

```diff
@@ -241,7 +316,17 @@ true
 ##### SET OPERATIONS #####
 
 Base.isempty(i::AbstractInterval) = LeftEndpoint(i) > RightEndpoint(i)
-Base.in(a, b::AbstractInterval) = !(a ≫ b || a ≪ b)
+function Base.in(a, b::AbstractInterval)
+    if isnothing(a)
+        if isnothing(first(b)) && isnothing(last(b))
+            return true
+        else
+            return false
+        end
+    else
+        return !(a ≫ b || a ≪ b)
+    end
```

Comment:
I would expect there should be no changes here and this logic would be handled by `less_than_disjoint`